src/std/text.ab

```diff
@@ -100,6 +100,62 @@ pub fun contains(text: Text, phrase: Text): Bool {
   return result == "1"
 }
 
+/// Checks if an array value is in the text.
+pub fun contains_any(text: Text, terms: [Text]): Bool {
+    for term in terms {
+        if contains(text, term) {
+            return true
+        }
+    }
+
+    return false
+}
+
+/// Checks if all the arrays values are in the string
+pub fun contains_all(text: Text, terms: [Text]): Bool {
+    for term in terms {
+        if not contains(text, term) {
+            return false
+        }
+    }
+
+    return true
+}
+
+/// Search all occurences of a regex pattern in the content.
+///
+/// Function uses `sed`
+pub fun contains_regex(source: Text, search: Text, extended: Bool = false): Bool {
```

Comment:
@Mte90 I'd change the name to `match_regex` because that's what it does. It tests if the regex matches anything and returns a boolean. It seems that it's a standard way to call this function. A name that contains `regex` (or any other variant of this name) and `match`:

| Language   | Method name          |
|------------|----------------------|
| JavaScript | `RegExp.test`        |
| Python     | `re.match`           |
| Rust       | `Regex::is_match`    |
| Go         | `regexp.MatchString` |
| C++        | `std::regex_match`   |
| Java       | `String.matches`     |
| PHP        | `preg_match`         |